hw/mcu/dialog/da1469x/syscfg.yml

```diff
@@ -392,3 +403,4 @@ syscfg.restrictions:
     - "!SPI_1_MASTER || (SPI_1_MASTER_PIN_SCK && SPI_1_MASTER_PIN_MOSI && SPI_1_MASTER_PIN_MISO)"
     - "!SPI_0_SLAVE || (SPI_0_SLAVE_PIN_SCK && SPI_0_SLAVE_PIN_MOSI && SPI_0_SLAVE_PIN_MISO && SPI_0_SLAVE_PIN_SS)"
     - "!SPI_1_SLAVE || (SPI_1_SLAVE_PIN_SCK && SPI_1_SLAVE_PIN_MOSI && SPI_1_SLAVE_PIN_MISO && SPI_1_SLAVE_PIN_SS)"
+    - "PLL_ON || MCU_SYSCLK_SOURCE != PLL96"
```

Comment:
should be `MCU_PLL_ON` although I'd change that name to `MCU_PLL_ENABLE` (like `MCU_DCDC_ENABLE`)